common/src/Polyhedron_Clip.h

```diff
@@ -47,24 +47,51 @@ typename Polyhedron<T,FP,VP>::ClipResult Polyhedron<T,FP,VP>::clip(const Plane<T
     if (!vertexResult.success())
         return vertexResult;
     
-    // Now we know that the brush will be split.
     // The basic idea is now to split all faces which are intersected by the given plane so that the polyhedron
     // can be separated into two halves such that no face has vertices on opposite sides of the plane.
-    const Seam seam = intersectWithPlane(plane, callback);
-    
-    // We construct a seam along those edges which are completely inside the plane and delete the half of the
-    // polyhedron that is above the plane. The remaining half is an open polyhedron (one face is missing) which
-    // is below the plane.
-    split(seam, callback);
+    // Sometimes building a seam fails due to floating point imprecisions. In that case, intersectWithPlane
+    // throws a GeometryException which we catch here.
+    try {
+        const Seam seam = intersectWithPlane(plane, callback);
+
+        // We construct a seam along those edges which are completely inside the plane and delete the half of the
+        // polyhedron that is above the plane. The remaining half is an open polyhedron (one face is missing) which
+        // is below the plane.
+        split(seam, callback);
+        
+        // We seal the polyhedron by creating a new face.
+        sealWithSinglePolygon(seam, callback);
+        updateBounds();
+        
+        assert(checkInvariant());
+        return ClipResult(ClipResult::Type_ClipSuccess);
+    } catch (const NoSeamException&) {
+        return ClipResult(ClipResult::Type_ClipUnchanged);
```

Comment:
Is it possible that `ClipResult::Type_ClipEmpty` is the right answer to return instead?
Might need to find the furthest vertex from the plane, and if it's in front, return Type_ClipUnchanged, otherwise return `ClipResult::Type_ClipEmpty`. I think this can be a future fix / enhancement though.